github/apps_installation.go

```diff
@@ -47,3 +50,35 @@ func (s *AppsService) ListRepos(ctx context.Context, opt *ListOptions) ([]*Repos
 
 	return r.Repositories, resp, nil
 }
+
+// AddRepository adds a single repository to an installation.
+//
+// GitHub API docs: https://developer.github.com/v3/apps/installations/#add-repository-to-installation
+func (s *AppsService) AddRepository(ctx context.Context, instID, repoID int) (*Repository, *Response, error) {
+	u := fmt.Sprintf("/apps/installations/%v/repositories/%v", instID, repoID)
```

Comment:
We need to not include the starting slash in the URLs, otherwise enterprise GH URLs will not work.

This should be:

```Go
u := fmt.Sprintf("apps/installations/%v/repositories/%v", instID, repoID)
```